content/blog/entries/cc-vocabulary-the-main-course/contents.lr

```diff
@@ -50,105 +52,32 @@ brainchild comes of age.
 
 **Presenting to you our special menu of Vue components.**
 
-<table>
-<tbody>
-    <tr>
-        <th colspan="2"><h3>Tokens</h3></th>
-    </tr>
-    <tr>
-        <td>Colors</td>
-        <td><img src="colors.png"></td>
-    </tr>
-    <tr>
-        <td>Fonts</td>
-        <td><img src="fonts.png"></td>
-    </tr>
-    <tr>
-        <td>Spaces</td>
-        <td><img src="spaces.png"></td>
-    </tr>
-    <tr>
-        <th colspan="2"><h3>Elements</h3></th>
-    </tr>
-    <tr>
-        <td>Button</td>
-        <td><img src="button.png"></td>
-    </tr>
-    <tr>
-        <td>InputField</td>
-        <td><img src="inputfield.png"></td>
-    </tr>
-    <tr>
-        <td>SelectField</td>
-        <td><img src="selectfield.png"></td>
-    </tr>
-    <tr>
-        <td>Heading</td>
-        <td><img src="heading.png"></td>
-    </tr>
-    <tr>
-        <td>Paragraph</td>
-        <td><img src="paragraph.png"></td>
-    </tr>
-    <tr>
-        <td>LicenseBadge</td>
-        <td><img src="licensebadge.png"></td>
-    </tr>
-    <tr>
-        <td>LicenseIconography</td>
-        <td><img src="licenseiconography.png"></td>
-    </tr>
-    <tr>
-        <td>ProgressBar</td>
-        <td><img src="progressbar.png"></td>
-    </tr>
-    <tr>
-        <td>Shield</td>
-        <td><img src="shield.png"></td>
-    </tr>
-    <tr>
-        <th colspan="2"><h3>Layouts</h3></th>
-    </tr>
-    <tr>
-        <td>Container</td>
-        <td><img src="container.png"></td>
-    </tr>
-    <tr>
-        <td>Grid</td>
-        <td><img src="grid.png"></td>
-    </tr>
-    <tr>
-        <td>Table</td>
-        <td><img src="table.png"></td>
-    </tr>
-    <tr>
-        <th colspan="2"><h3>Patterns</h3></th>
-    </tr>
-    <tr>
-        <td>Header</td>
-        <td><img src="header.png"></td>
-    </tr>
-    <tr>
-        <td>Footer</td>
-        <td><img src="footer.png"></td>
-    </tr>
-    <tr>
-        <td>Locale</td>
-        <td><img src="locale.png"></td>
-    </tr>
-    <tr>
-        <td>Hello</td>
-        <td><img src="hello.png"></td>
-    </tr>
-    <tr>
-        <th colspan="2"><h3>Templates</h3></th>
-    </tr>
-    <tr>
-        <td>Index</td>
-        <td><img src="orange.png" onclick="this.src = 'green.png'"></td>
-    </tr>
-</tbody>
-</table>
+|                    | Tokens                      |
+| ------------------ | :-------------------------- |
```

Comment:
The token heading is on the same level as `**Elements**`, `**Layouts**` and `**Patterns**` below so making it the table title isn't right.
```suggestion
| Component          | Image                       |
| ------------------ | :-------------------------- |
| **Elements**       |                             |
```